CONTRIBUTING.md

```diff
@@ -103,25 +103,78 @@ working on the issue.
 
 ### Dataset research
 
-### Algorithm research
+Please follow the below steps if your issue is about researching and/or
+analyzing a dataset:
 
-<!-- The PR checklist:
+1. In your local copy of the repository, create a branch to take care of the issue
+using the following naming convention for the branch:
+*[naming convention missing]*
+2. Work in your branch in the `data` folder until you have finished your research
```

Comment:
For step 2, I think it should be in the notebooks folder, instead of data. In data folder I don't think there should be code, just documentation on the datasets we are using and information on how to request it (no done yet).

Reply:
So you mean dataset research is also done with Jupyter?

Reply:
The data folder confuses me I think haha

Reply:
yes! normally, the data folder would contain the dataset, but because of GDPR we cannot publish ours. Which is why the data folder will only contain instructions how to request it and aggregated info about our datasets + research into labeling methodologies for similar datasets (which is what we currently have). 

We can also this discuss if it's necessary at all to have this folder on our repo, although we should definitely mention in our instructions that once people receive these datasets, they should put them in this folder in order for the notebooks/modeling code to work. This way our work should be consistent and we would probably avoid errors related to paths.

Reply:
Aaaah ok I got it now! So I think we don't need to split the research issues into dataset/algorithm, I will do only one "Working on a research issue" section.